test/blackbox/BlackboxTestsLivelinessQos.cpp

```diff
@@ -32,7 +32,7 @@ TEST(LivelinessQos, Liveliness_Automatic_Reliable)
     // Liveliness lease duration and announcement period
     uint32_t liveliness_ms = 200;
     Duration_t liveliness_s(liveliness_ms * 1e-3);
-    Duration_t announcement_period(liveliness_ms * 1e-3 * 0.5);
+    Duration_t announcement_period(liveliness_ms * 1e-3 * 0.01);
```

Comment:
As this is just a backport, I guess it is fine to leave these timings, although they are not enough to make tests stable.

Reply:
Outdated dut to #689 backport.